src/test/java/mate/academy/dao/impl/UserDaoImplTest.java

```diff
@@ -35,14 +35,14 @@ void setUp() {
     }
 
     @Test
-    void save_Ok() {
+    void save_correctData_Ok() {
```

Comment:
would be nice if you'd be more specific on the condition like `validUser`, `validRole` etc.